app/views/catalog_record_ids/_edit.html.erb

```diff
@@ -24,17 +24,21 @@
         <% end %>
 
         <template data-nested-form-target='template'>
-          <%= form.fields_for :catkeys, CatkeyForm::Row.new, child_index: 'TEMPLATE_RECORD' do |catkey_form| %>
-            <%= render 'row', form: catkey_form %>
+          <%= form.fields_for :catalog_record_ids, CatalogRecordIdForm::Row.new, child_index: 'TEMPLATE_RECORD' do |catalog_record_id_form| %>
+            <%= render 'row', form: catalog_record_id_form, hidden: false %>
```

Comment:
Because I added `hidden` to allow writing Folio instance HRIDs to the form invisibly when in Symphony mode (and vice versa). This is the stuff we don't want hidden.